### DISCUSSION TITLE
Add some

### DISCUSSION THREAD

**Discussion by DJChrisRey** [mampfes/hacs_waste_collection_schedule#1415] [Ideas]
Hallo can you add german / Northeim https://abfall.nerdbridge.de/

Its an ics from people who need it and it was fine to add it

**Comment by DJChrisRey** [comment]
its a axemple https://abfall.nerdbridge.de/ical/abfall-nom-ellierode_hardegsen-HM2W,PET,BIO,PAP,GL.ics

**Comment by 5ila5** [comment]
already supported
README.md:
[
Landkreis Northeim (unofficial)](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/ics/nerdbridge_de.md) / abfall.nerdbridge.de
